File: Medical-Invertory-Management/src/main/java/com/devansh/Medical/Invertory/Management/models/SalesHistory.java
package com.devansh.Medical.Invertory.Management.models;


import com.fasterxml.jackson.annotation.JsonIdentityInfo;
import com.fasterxml.jackson.annotation.ObjectIdGenerators;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.stereotype.Component;


@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Entity
@Table(name = "salesHistory")
public class SalesHistory {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private int id;
    private String productName;
    private String userName;
    private int quantity;
    private double price;
    private boolean sold = false;
    private boolean purchased = false;
    private boolean deleted = false;
}

File: Medical-Inventory-User-Springboot/app/src/main/java/com/example/medicalinventoryuserspringboot/model/SalesHistory.kt
package com.example.medicalinventoryuserspringboot.model


data class SalesHistory(

    var id: Int = 0,

    var productName: String = "",
    var userName: String = "",
    var quantity: Int = 0,
    var price: Double = 0.0,
    var sold: Boolean = false,
    var purchased: Boolean = false,
    var deleted: Boolean = false
)